### ISSUE TITLE
Minimized MDI Child 

### ISSUE THREAD

**Issue by gMagnus87** [veselcraft/progman#3]
This is an issue for Windows itself, not Progman, maybe it can customize the behavior of the MDI Child instead of using the default by Windows.

I would love that the original behavior of the MDI Child could be recovered, since from W9X these become title bars when they are minimized, which not only does not coincide with the original but also makes it impossible to read the element correctly.

![](https://user-images.githubusercontent.com/28157324/81138446-80cbc580-8f38-11ea-9e9d-636881f195c6.png)

**Comment by Marfjeh** [comment]
I will have a look into this if i have some free time

**Comment by gMagnus87** [comment]
Maybe this helps, at the time I tried to use it but the truth is that I am not too good at understanding how other people's code works.

https://www.codeproject.com/Tips/823842/MDI-Forms-that-Minimize-to-Icons

![](https://www.codeproject.com/KB/graphics/823842/DemoApp.png)

However, it is not complete to give that W3X appearance, it would be necessary to add the name below the icon / image to the code

**Comment by Marfjeh** [comment]
Yes i've actually looked into that project. there was only a issue with it but i cant remember its some time ago i've worked on this haha.
